sdk/core/core-rest-pipeline/review/core-rest-pipeline.api.md

```diff
@@ -89,6 +89,16 @@ export function decompressResponsePolicy(): PipelinePolicy;
 // @public
 export const decompressResponsePolicyName = "decompressResponsePolicy";
 
+// @public
+export function defaultRetryPolicy(options?: DefaultRetryPolicyOptions): PipelinePolicy;
+
+// @public
+export interface DefaultRetryPolicyOptions {
```

Comment:
`PipelineRetryOptions` was added in a previous PR and I wonder if `DefaultRetryPolicyOptions` should be removed in favor of `PipelineRetryOptions`.

Reply:
Can do!

Reply:
We generally name the option parameters based on the function or class that they belong to. I could tie them together by making one an extension of the other, but I don’t think I should replace one with the other.

Reply:
I'm fine with `DefaultRetryPolicyOptions` extending `PipelineRetryOptions`, but since they're identical today I'd also be fine with just re-using `PipelineRetryOptions` and creating `DefaultRetryPolicyOptions` later once we have a unique option on it since extending an existing input interface with new optional members is not breaking.

Reply:
I’ll extend PipelineRetryOptions 🙂

Reply:
Updated here: https://github.com/Azure/azure-sdk-for-js/pull/19078/commits/e217fc5d46dc568209f8358d462736f04de03de0